src/App.js

```diff
@@ -20,55 +20,66 @@ function Board({ xIsNext, squares, onPlay }) {
     } else {
       nextSquares[i] = "O";
     }
-    onPlay(nextSquares);
+    onPlay(nextSquares, i);
   }
-  const winner = calculateWinner(squares);
+
+  const winnerData = calculateWinner(squares);
   let status;
-  if (winner) {
-    status = "Winner: " + winner;
+  if (winnerData && winnerData.winner) {
+    status = "Winner: " + winnerData.winner;
+  } else if (currentMove === 9 && !winnerData) {
+    status = "Draw";
   } else {
     status = "Next player: " + (xIsNext ? "X" : "O");
   }
   
-function Row(n) {
-  const row = [];
-  for(let i = n; i < n + 3; i++) {
-    row.push(<Square value={squares[i]} onSquareClick={() => handleClick(i)}/>);
+  function Row(n) {
+    const row = [];
+    let a, b, c;
+    if (winnerData) { [a, b, c]= winnerData.winnerLine;};
+ 
+    for(let i = n; i < n + 3; i++) {
+      if(winnerData && (i === a || i === b || i === c)) {
+        row.push(<Square className="winner" value={squares[i]} onSquareClick={() => handleClick(i)} />);
+      } else {
+        row.push(<Square value={squares[i]} onSquareClick={() => handleClick(i)} />);
+      }      
+    }
+    return row;
   }
-  return row;
-}
 
-function Rows(n) {
-  const rows = [];
-  for(let i = n; i <= 6; i += 3) {
-    rows.push(
-      <div className="board-row">
-        {Row(i)}
-      </div>)
+  function Rows() {
+    const rows = [];
+    for(let i = 0; i <= 6; i += 3) {
+      rows.push(
+        <div className="board-row">
+          {Row(i)}
+        </div>)
+    }
+    return rows;
   }
-  return rows;
-}
 
-return (
-  <>
-    <div className="status">{status}</div>
-    {Rows(0)}
-  </>
-)
+  return (
+    <>
+      <div className="status">{status}</div>
+      {Rows()}
+    </>
+  )
 }
 
 export default function Game() {
   const [history, setHistory] = useState([Array(9).fill(null)]);
   const [currentMove, setCurrentMove] = useState(0);
   const [isAscending, setOrder] = useState(true);
-  const [historyOrder, setHistoryOrder] = useState(history.slice());
+  const [indexes, setIndexes] = useState([]);
   const xIsNext = currentMove % 2 === 0;
   const currentSquares = history[currentMove];
 
-  function handlePlay(nextSquares) {
+  function handlePlay(nextSquares, i) {
```

Comment:
Change the variable 'i' to be more descriptive.

Reply:
Done.